lib/fathead/opam/parse.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/python
+
+import os
+from urlparse import urljoin
+
+from bs4 import BeautifulSoup
+
+__target__ = "https://opam.ocaml.org/"
+__module__ = "OPAM"
+
+class Environment(object):
+  """
+  The Environment class is responsible for setting up the directory
+  and retrieving the appropriate files from the web. It is made up of
+  a constructor and a 'setup/0' and 'get_contents/0' member functions.
+  """
+  def __init__(self):
+    print("Environment nstantiated")
```

Comment:
Opps! Must've missed that one. I reset the head and pushed with much farse incase this is missing. 😉